gen/docdb

```diff
@@ -36,11 +36,11 @@ my $outputData = {
 						 members => [],
 					 };
 
-foreach my $resourceData ($data->getChildElements) {
-	processResourceData $resourceData;
+foreach my $resourceId ($data->getResourceIds()) {
+	processResourceMembersData $resourceId;
 
-	foreach my $groupData (($resourceData->getChildElements)[0]->getChildElements) {
-		my $groupMembersLogins = processGroupData $groupData;
+	foreach my $groupId ($data->getGroupIdsForResource( resource => $resourceId )) {
+		my $groupMembersLogins = processGroupData $resourceId, $groupId;
```

Comment:
Variable `$groupMembersLogins` is not used, you can simply call `processGroups()` method without processing returned value.